Duplicati/UnitTest/UriUtilityTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿//  Copyright (C) 2018, The Duplicati Team
+//  http://www.duplicati.com, info@duplicati.com
+//
+//  This library is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as
+//  published by the Free Software Foundation; either version 2.1 of the
+//  License, or (at your option) any later version.
+//
+//  This library is distributed in the hope that it will be useful, but
+//  WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+//  Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public
+//  License along with this library; if not, write to the Free Software
+//  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+
+using NUnit.Framework;
+using System.Collections.Specialized;
+
+namespace Duplicati.UnitTest
+{
+    public class UriUtilityTests
+    {
+        [Test]
+        [Category("UrlUtility")]

```

Comment:
The Travis configuration requires that we explicitly add each test category to the build.  Would you mind modifying the [travis.yml](https://github.com/duplicati/duplicati/blob/master/.travis.yml) file to include the `UrlUtility` category?